test-cases/longevity/longevity-twcs-3h.yaml

```diff
@@ -3,8 +3,8 @@ bench_run: true
 
 stress_cmd: ["scylla-bench -workload=timeseries -mode=write -replication-factor=3 -partition-count=400 -clustering-row-count=10000000 -clustering-row-size=200 -concurrency=100 -rows-per-request=100  -connection-count 100 -max-rate 50000 --timeout 120s -duration=170m"]
 stress_read_cmd: [
-    "scylla-bench -workload=timeseries -mode=read -partition-count=20000 -concurrency=100 -replication-factor=3 -clustering-row-count=10000000 -clustering-row-size=200  -rows-per-request=100 -start-timestamp=SCT_TIME -write-rate 125 -duration=10m -distribution hnormal --connection-count 100 -duration=170m",
-    "scylla-bench -workload=timeseries -mode=read -partition-count=20000 -concurrency=100 -replication-factor=3 -clustering-row-count=10000000 -clustering-row-size=200  -rows-per-request=100 -start-timestamp=SCT_TIME -write-rate 125 -duration=10m -distribution uniform --connection-count 100 -duration=170m"
+    "scylla-bench -workload=timeseries -mode=read -partition-count=20000 -concurrency=100 -replication-factor=3 -clustering-row-count=10000000 -clustering-row-size=200  -rows-per-request=100 -start-timestamp=GET_WRITE_TIMESTAMP -write-rate 125 -duration=10m -distribution hnormal --connection-count 100 -duration=170m",
```

Comment:
We need to do the same also for "write_rate" it's also a calculated value.
Although here it's easier because it's not something that changes and I think that the formula is "max_rate" / "partition_count".

Reply:
As Avi suggest in comment: https://github.com/scylladb/scylla/issues/8030#issuecomment-1005017695 the -write-rate could be counted as max-rate * expected delay = 50k * 10ms = 500
But according your formula write rate is already have valid value: max-rate / partition_cound = 50K / 400 = 125

I set medium from both results: 250.

Reply:
AFAIU, The write-rate here is just a hint for the read workload so it won't run the reads too fast and reach to points where the write workload hasn't written yet.
It has nothing to do with what Avi asked. for that we set the parallelism and the max-rate.

Reply:
Also, I didn't calculated it, I got it from the write workload output, same as with the timestamp.

Reply:
i added comments to config yaml. How it is correctly should be calculated.